Clients/GUI/CustomValidationAtributes/NameValidation.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using GUI.ViewModels;
+
+namespace GUI.CustomValidationAtributes
+{
+    public class NameValidation : ValidationAttribute
+    {
+        private string name;
+
+        public NameValidation(string name)
+        {
+            this.name = name;
+        }
+
+        public string GetErrorMessageNoNumbers() =>
+            $"Not allow number in {name}";
```

Comment:
May be using not input name, but a word "name" would be more understandable in error messages?